packages/demo-app/src/app/demos/theme/useTheme.ts

```diff
@@ -1,6 +1,6 @@
-import * as React from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
The selling point of * is to avoid doing back and forth with the top of the file when working the logic. It's similar to what guys with Python are doing, they import * all the modules. Save time, makes git diff less noisy.

Considering that React can't be tree-shaken, I don't see the value of using named imports. Any pragmatic reasons?